src/components/ui/BlockQuote/BlockQuote.tsx

```
@@ -9,17 +9,29 @@ export type BlockQuoteProps = {
     children: React.ReactNode;
     customRootClass?: string;
     className?: string;
-    color?:string;
+    color?: string;
+    variant?: string;
+    size?: string;
     props?: Record<string, any>[]
 }
-const BlockQuote = ({ children, customRootClass = '', className = '', color = '', ...props }: BlockQuoteProps) => {
+const BlockQuote = ({ children, customRootClass = '', className = '', color = '', variant= '', size= '', ...props }: BlockQuoteProps) => {
```

ESLint

[error] Operator '=' must be spaced.

(space-infix-ops)